libs/shared/src/types/clerk/types.ts

```diff
@@ -0,0 +1,21 @@
+import { ProductUseCases } from '../../dto';
+import { ApiServiceLevelEnum } from '../organization';
+import { IServicesHashes } from '../../entities/user';
+import { JobTitleEnum } from '../organization';
+
+export type _OrganizationPublicMetadata = {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** What is the intention behind prefixing these with an underscore? It's a new naming pattern we are introducing and I think we should be careful with such decisions. If there isn't a solid reason, I suggest we leave them unprefixed.‏

Reply:
Perfect!